satellite/metainfo/metainfo.go

```diff
@@ -523,15 +523,22 @@ func (endpoint *Endpoint) filterValidPieces(ctx context.Context, pointer *pb.Poi
 			return Error.New("all pieces needs to have the same size")
 		}
 
-		// we repair when the number of healthy files is less than or equal to the repair threshold
-		// except for the case when the repair and success thresholds are the same (a case usually seen during testing)
+		// We repair when the number of healthy files is less than or equal to the repair threshold
```

Comment:
This comment is bogus because this code isn't called by repair (anymore?)

Reply:
This code was never called by repair afaik. The comment seems to clarify why there is a `&&` for the error check